sdk/lib/core/uri.dart

```diff
@@ -459,21 +499,23 @@ abstract interface class Uri {
   /// The returned scheme is canonicalized to lowercase letters.
   String get scheme;
 
-  /// The authority component.
+  /// The authority (hostname) component.
```

Comment:
I wouldn't add the `(hostname)` comment here at all.

This is the *authority* accessor, which explicitly accesses *more* than just `.host`.